CONTRIBUTING.md

```diff
@@ -17,3 +17,32 @@ rover is an open source project developed by [Apollo GraphQL](https://apollograp
 - One approval from a member of the Apollo tooling team.
   - The team consists of @jbaxleyiii, @ashleygwilliams, @JakeDawkins, and @EverlastingBugstopper.
   - PRs with changes in `docs/` automatically add @StephenBarlow as a reviewer. PRs are not blocked on this approval.
+
+## Adding new commands/subcommands
+
+When adding a new command or a subcommand, you should make sure that the field in the new command struct is titled `command`. This allows our automated anonymous usage collection to work properly.
+
+Should you add arguments that are _not_ subcommands, the arguments you specify will automatically be reported to Apollo's Telemetry backend, unless you specify that an argument should be ignored with `#[serde(skip_serializing)]`. You should be careful to add the `skip_serializing` attribute to any fields that may contain data that may not be anonymous. This can be anything from a profile name to a path on the user's computer that could include their name.
```

Comment:
"Should" -> "If"

"will automatically be reported" -> "are automatically reported"

"You should be careful to" -> "Make sure you"

"that may contain data that may not be anonymous..." -> "that might contain data that isn't anonymous (such as a username or a file path that might include a username)."